src/io/rme_result_io.jl

```diff
@@ -169,23 +170,25 @@ groupings for compatibility with ADRIA scenario plotting.
 function _construct_scenario_groups(
     inputs::DataFrame,
     n_scenarios::Int
-)::Dict{Symbol, BitVector}
+)::Dict{Symbol,BitVector}
     if size(inputs) == (0, 0)
         return Dict(:counterfactual => BitVector([true for _ in 1:n_scenarios]))
     end
 
     counterfactual_scens::BitVector = BitVector([
         p_a == 0.0 && e_a == 0 for (p_a, e_a)
-            in zip(inputs.outplant_area_pct, inputs.enrichment_area_pct)
+        in
+        zip(inputs.outplant_area_pct, inputs.enrichment_area_pct)
```

Comment:
I think a better way of doing this would be:

```
inputs.outplant_area_pct .== 0.0 .&&  inputs.enrichment_area_pct .== 0
```

I know, this is not just formatting. Maybe we can ignore this for now, but I just wanted to say that I really don't like this formatting (but again, I'm happy to ignore it).